.env.example

```diff
@@ -1,2 +1,3 @@
 BUCKET_NAME_PREFIX="my-website-honestly"
 BUCKET_NAME_SUFFIX="dev"
+API_KEY='my-website-honestly'
```

Comment:
have clarified on slack that this is for the API_KEY name and definitely not the key itself. Going to move it inline as it's not a secret.